webpack.config.js

```diff
@@ -125,13 +125,18 @@ module.exports = {
   resolve: {
     extensions: ['.webpack.js', '.web.js', '.ts', '.tsx', '.js', '.json', '.css', '.scss'],
     alias: {
-      Configuration: path.resolve(configurationPath)
+      Configuration: path.resolve(configurationPath),
+      jsonld$: path.resolve(__dirname, './node_modules/jsonld/dist/jsonld.min.js')
     },
     modules:  [
       path.join(__dirname, "src"),
       "node_modules"
     ],
   },
+  
+  node: {
+    net: 'empty',
```

Comment:
The joi dependency in poet-js was causing this error: 
`Module not found: Error: Cannot resolve module 'net' on node_modules/joi/lib @ ./~/joi/lib/string.js 3:10-24`
I was able to fix this locally with this line, which I got from the joi issues: https://github.com/hapijs/joi/issues/665#issuecomment-113713020

Reply:
uhmm because `joi` is not prepared to use in the front end. For this reason, I saw in several projects are using https://github.com/jquense/yup instead of `joi`